File: src/main/java/checkdigitcli/command/Exit.java
package checkdigitcli.command;

import checkdigitcli.main.CheckDigitCLI;
import picocli.CommandLine.Command;

@Command(name = "exit", aliases = { "quit", "terminate" })
public class Exit implements Runnable {

	@Override
	public void run() {
		System.out.println("Exiting...");
		CheckDigitCLI.setTerminating(true);
	}
}

File: src/main/java/checkdigitcli/command/Exit.kt
package checkdigitcli.command

import checkdigitcli.main.CheckDigitCLI
import picocli.CommandLine

@CommandLine.Command(name = "exit", aliases = ["quit", "terminate"])
class Exit : Runnable {
    override fun run() {
        println("Exiting...")
        CheckDigitCLI.isTerminating = true
    }
}